package.json

```diff
@@ -38,5 +38,8 @@
   },
   "overrides": {
     "rollup": "3.2.5"
+  },
+  "dependencies": {
+    "markdown-it": "^13.0.2"
```

Comment:
Shouldn't this be in `burno-app` package.json only?

Reply:
I'm not sure how it should really be if you know, correct me.